process-automation/latest/model/forms.md

```diff
@@ -186,6 +186,32 @@ The advanced properties for a REST dropdown field are:
 | Path to array in JSON response | The path to the JSON response. Enter `.` to use the full path. |
 | ID property | The ID of the REST service. |
 | Label property | The name of the REST service. |
+| Conditional | Turn this option on if you would like to link your drop down list with another drop down list and to create a conditional relationship between them. For example, if you select a country from one drop down list, the second drop down list will only show cities from that country. |
```

Comment:
should we use `dropdown field` or `dropdown widget` instead of `drop down list`?

Reply:
Changed